### ISSUE TITLE
Scroll button does not work

### ISSUE THREAD

**Issue by coreline** [FreeRDP/FreeRDP#2949]
Scroll up/down works correctly but middle button on the scroll does not work.

**Comment by bmiklautz** [comment]
@coreline please have a look to https://github.com/FreeRDP/FreeRDP/wiki/BugReporting and add more information.

**Comment by coreline** [comment]
I do the following: 
1. Connecting to 7 win / win 2008 
2. I opened the Google Chome 
3. Open multiple tabs on Google Chome 
4. Press one of the tabs on the middle mouse button. I expect that the tab should be closed, but not locked. 
5. On some website click the middle mouse button. The mouse cursor should take the form of a cross. If you keep the mouse up or down, the page will start soon scrolls. But none of this is happening.

**Comment by bmiklautz** [comment]
@coreline what version of freerdp are you using?
I've tested the above's procedure with the latest freerdp-nightly against windows 7 and the wheel button on my mouse worked fine.

**Comment by nicoulaj** [comment]
I have a similar issue, I can scroll up but not down with the mouse wheel.

System: Arch Linux (linux 4.4.5 x86_64)

Freerdp version: `2.0.0-dev (git 75ae3f5)` (from arch linux user repository package `freerdp-git`, same issue with stable package `freerdp 1.2.0`)

Command:
```
xfreerdp /v:$HOST \
         /gd:$GATEWAY \
         /gu:$USER \
         /gp:$PASSWORD \
         /u:$USER \
         /g:$GATEWAY \
         /w:1920 \
         /h:1020 \
         /cert-ignore \
         +grab-keyboard \
         +clipboard \
         -themes \
         -wallpaper \
         /gdi:hw \
         /gfx \
         +gfx-h264 \
         +gfx-thin-client \
         +gfx-small-cache \
         +gfx-progressive
```

It still happens if I remove all the non-mandatory options:
```
xfreerdp /v:$HOST \
         /gd:$GATEWAY \
         /gu:$USER \
         /gp:$PASSWORD \
         /u:$USER \
         /g:$GATEWAY \
         /w:1920 \
         /h:1020 \
         /cert-ignore
```

Logs don't show anything particular.
Here is a full log with `TRACE` level with connexion + attempt to scroll down: https://gist.github.com/nicoulaj/ab5fce5f2b9d44968c26

**Comment by akallabeth** [comment]
@nicoulaj Working fine here with debian jessie. Do you have some kind of button remapping active?

**Comment by nicoulaj** [comment]
No, I don't, and I tested several mices and several desktop environments on client (GNOME 3, cinnamon, XCFE) and server side (GNOME 2, XFCE).

I'm on Arch Linux by the way (CentOS 6 on server side).

**Comment by pachuco** [comment]
Confirming issue with
Host: Windows POSReady 2009(XP SP3 derivate) x86
Client: Windows 8 x86

Currently using 2.0.0-rc1 nightly.

**Comment by bmiklautz** [comment]
@pachuco this bug is for x11.

**Comment by bmiklautz** [comment]
This issue should be fixed with the latest master (upcoming v2.0). Please re-open or create a new issue if you still encounter this problem.
